src/models/base/anthropic_model.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, config: ModelConfig):
         super().__init__(config)
 
         self.model = config.name
+        PromptStatistics.set_model(config.name)
```

Comment:
These are separate as reset is done as part of the benchmark. This is because there can be multiple benchmark that use the same model, so we reset multiple times.